media-libs/openexr/openexr-2.2.1.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools multilib-minimal
+
+DESCRIPTION="ILM's OpenEXR high dynamic-range image file format libraries"
+HOMEPAGE="http://openexr.com/"
+SRC_URI="http://download.savannah.gnu.org/releases/openexr/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/23" # based on SONAME
+KEYWORDS="~amd64 -arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~x64-macos ~x86-solaris"
+IUSE="cpu_flags_x86_avx examples static-libs"
+
+RDEPEND="
+	sys-libs/zlib[${MULTILIB_USEDEP}]
+	>=media-libs/ilmbase-${PV}:=[${MULTILIB_USEDEP}]"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig[${MULTILIB_USEDEP}]
+	>=sys-devel/autoconf-archive-2016.09.16"
+
+RESTRICT="test"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-2.2.0-fix-cpuid-on-abi_x86_32.patch"
```

Comment:
just a suggestion, the more patches you accumulate, the better it is to group them like `# xx branch` / `# master` / `# TODO upstream` to keep track of them, not least to help the next person looking at it.

Reply:
I think all patches are official upstream patches in their develop branch (that's what they call master). It's a good idea for the comment so I will add that. :) Thanks for your review.